runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -344,6 +344,12 @@ public void onChatMessage(ChatMessage event)
 			String fileName = "Kill " + format(new Date());
 			takeScreenshot(fileName);
 		}
+
+		if (config.screenshotBossKills() && chatMessage.startsWith("Your") && chatMessage.contains("kill count is"))
```

Comment:
This should use `BOSSKILL_MESSAGE_PATTERN` instead of `chatMessate.startsWith` and `chatMessage.contains`.